apps/dynamo/lib/dynamo/vnode_master.ex

```diff
@@ -45,6 +45,13 @@ defmodule Vnode.Master do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_cast({:repl_command, sender, msg}, state) do
+    Logger.info("Received command #{inspect(msg)} from #{sender}")
+    GenServer.cast(Node.self(), msg)
```

Comment:
I don't thinks this will be sent to an appropriate vnode. I should be sent vnode's pid not `Node.self()`